### PR TITLE
Magic numbers animation

### LINKED ISSUE #3038: Standardize view animation duration
If you check the usages of `animateView` you will see that the typical range will be 0 all the way to 500, with most in the 100-250 range.

It should've been done with a default value in the first place. It is pretty much arbitrary right now, it was whatever the person felt like at the time of writing.

I'll leave this task to another PR, which the goal will be to standardize the used duration of animations.

_Originally posted by @mauriciocolli in https://github.com/_render_node/MDE3OlB1bGxSZXF1ZXN0UmV2aWV3MzQxNTczMDc1/pull_request_reviews/more_threads_

### PR DESCRIPTION
**Pull request by Novmbrain**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Define two constants in `app/src/main/java/org/schabi/newpipe/util/Constants.kt`:

```kotlin
const val DEFAULT_LONG_ANIM_DURATION = 1500
const val DEFAULT_SHORT_ANIM_DURATION = 250
```

- Standardize the used duration of animations by replacing the magic number(typical range between 0 to 500) with these two constants in these following files:

```
main\java\org\schabi\newpipe\fragments\list\BaseListFragment.java
main\java\org\schabi\newpipe\local\BaseLocalListFragment.java
main\java\org\schabi\newpipe\fragments\detail\VideoDetailFragment.java
main\java\org\schabi\newpipe\fragments\list\channel\ChannelFragment.java
main\java\org\schabi\newpipe\fragments\list\playlist\PlaylistFragment.java
main\java\org\schabi\newpipe\fragments\list\search\SearchFragment.java
main\java\org\schabi\newpipe\info_list\holder\StreamMiniInfoItemHolder.java
main\java\org\schabi\newpipe\local\holder\LocalPlaylistStreamItemHolder.java
main\java\org\schabi\newpipe\local\holder\LocalStatisticStreamItemHolder.java
main\java\org\schabi\newpipe\local\playlist\LocalPlaylistFragment.java
````

#### Fixes the following issue(s)

- Fixes #3038 

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/player/Player.java [hunk 1]
```diff
@@ -235,7 +235,7 @@
     public static final String IS_MUTED = "is_muted";
 
     /*//////////////////////////////////////////////////////////////////////////
-    // Time constants
+    Time constants
```
Comment (XiangRongLin) [inline app/src/main/java/org/schabi/newpipe/player/Player.java:238]: With this change the comment is the odd one out if you compare it to the other ones

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:7.0.3'
```
Comment (XiangRongLin) [inline build.gradle:11]: Is updating dependencies intended?

File: app/src/main/java/org/schabi/newpipe/player/Player.java (hunk 1 above)
Comment (Novmbrain) [reply app/src/main/java/org/schabi/newpipe/player/Player.java:238]: Ok, sorry for this mistake, I rollbacked this change to keep the form of commentaire same

File: build.gradle (hunk 2 above)
Comment (Novmbrain) [reply build.gradle:11]: No, the update of the dependencies was not intended, we just want to keet the same version of gradle as upstream
[https://github.com/TeamNewPipe/NewPipe/blob/4fc9443b4fb2ec481f9d9bcd9b0df045f928307d/build.gradle](url)

Comment (TiA4f8R) [reply build.gradle:11]: If you want to do this, update your git branch from our upstream directly and don't replicate changes into yours. To do so, you will have to:

- add our repo as a git remote (in my commands, it is called `NEWPIPE_REMOTE_NAME`);
- fetch this remote (`git fetch NEWPIPE_REMOTE_NAME`);
- select your branch, drop the first commit (to remove this change) (you can do this by using `git rebase -i HEAD~NUMBER_OF_YOUR_COMMITS` and following instructions to remove the commit (you will need to put a `d` and a space before the SHA1 of the commit you want to remove then to save the commit message and to close your editor, if I am not wrong));
- rebase your branch into our `dev` branch (when your branch is selected, run `git rebase NEWPIPE_REMOTE_NAME/dev`);
- fix eventual conflicts and continue rebase;
- check if everything works as intended after the rebase;
- force push your changes your remote branch (the command line should be `git push -f origin magic_numbers_animation` in your case).

Comment (XiangRongLin) [reply build.gradle:11]: @Novmbrain Could you remove these unrelated changes from upstream? Either like described or some other method.

**Comment by XiangRongLin** [comment]
@Novmbrain Are you still working on this? If not I'm going to close this PR so other people can work on it.

**Pull request closed without merging**